x-pack/plugins/enterprise_search/public/applications/applications/components/engine/engine_connect/search_application_documentation.tsx

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiIcon,
+  EuiImage,
+  EuiLink,
+  EuiPanel,
+  EuiSpacer,
+  EuiText,
+  EuiTitle,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import welcomeGraphicLight from '../../../../../assets/images/welcome_light.svg';
+
+export const SearchApplicationDocumentation = () => {
+  return (
+    <EuiPanel color="transparent">
+      <EuiFlexGroup justifyContent="spaceBetween" direction="row">
+        <EuiFlexGroup direction="column">
+          <EuiImage
+            size="l"
+            float="left"
+            src={welcomeGraphicLight}
+            alt={i18n.translate(
```

Comment:
I think this could be considered a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/), which would mean we should set this to `""`. [I recently learned that.](https://github.com/elastic/kibana/pull/157021#discussion_r1189635209)

Reply:
There is another [instance](https://github.com/elastic/kibana/blob/main/x-pack/plugins/enterprise_search/public/applications/shared/add_content_empty_prompt/add_content_empty_prompt.tsx#L104) in where we use this image. We should probably add this change there as well.